### DISCUSSION TITLE
Why most of the necessary buttons are greyed out? There's no way to accept [left to right] only [right to left] available.

### DISCUSSION THREAD

**Discussion by hcchengithub** [WinMerge/winmerge#670] [Q&A]
I tried to open this sample conflict file hoping WinMerge help to do the merge 

```
<<<<<<< HEAD
        11
=======
        22
>>>>>>> projectX
<<<<<<< HEAD
        33
=======
        44
        44
>>>>>>> projectX
```

But I found most buttons are greyed out so why a merge tool refuse to do those ? or how ? 
![image](https://user-images.githubusercontent.com/1297015/110412122-a2e92380-80c6-11eb-95cc-9d3c2ab2406c.png)

**Comment by sdottaka** [comment]
When you open a conflict file, the left side becomes read-only and you can edit the file on the right side.

Therefore, you cannot copy blocks from right to left, and the "Copy Left" icon will be Disabled.
Also, when you look at the screenshot, the cursor position is outside the Diff block, so the "Copy Right" icon is also Disabled.

To make the "Copy Right" icon Enabled, move the cursor inside the Diff block.

**Reply by hcchengithub** [reply]
Thanks.

**Comment by hcchengithub** [comment]
WinMerge seems not for source code merge conflict resolving? That needs to 
- accept left
- accept right
- accept both

and probably remove both might be needed but rarely I believe. WinMerge does not do them?

**Reply by sdottaka** [reply]
When you open the conflict file, everything on the right side is in the "accept right" state from the beginning.
If you want to "accept left", press the "Copy Right" button.
If you want to "accept both", there is no button for it, so copy the text block in the left pane and paste it into the right pane.

**Reply by hcchengithub** [reply]
Ahhhhh so it's actually sufficient enough! 
Thank you very much.
